application/src/test/scala/com/azavea/franklin/api/services/SearchServiceSpec.scala

```diff
@@ -117,6 +120,51 @@ class SearchServiceSpec
     result.features.head.id should beEqualTo(stacItem.id)
   }
 
+  def find2ItemsWhenExpected = prop {
+    (stacItem1: StacItem, stacItem2: StacItem, stacCollection: StacCollection) =>
+      val resourceIO = testClient map {
+        _.getCollectionItemsResource(stacItem1 :: stacItem2 :: Nil, stacCollection)
+      }
+      val requestIO = resourceIO flatMap { resource =>
+        def getSearchCollection(searchFilters: SearchFilters): IO[Option[StacSearchCollection]] = {
+          val request =
+            Request[IO](method = Method.POST, uri = Uri.unsafeFromString(s"/search"))
+              .withEntity(searchFilters.asJson)
+          (for {
+            resp    <- testServices.searchService.routes.run(request)
+            decoded <- OptionT.liftF { resp.as[StacSearchCollection] }
+          } yield decoded).value
+        }
+
+        resource.use {
+          case (collection, _) =>
+            val inclusiveParams =
+              FiltersFor.inclusiveFilters(collection).copy(limit = NonNegInt.unsafeFrom(1).some)
```

Comment:
You can import `eu.timepit.refined.auto._` and safely use a literal `1` here if you want. Though I'm not sure how that will play with `.some` tbh

Reply:
replaced with `NonNegInt.from(1).toOption` shrug